core/src/datayoga_core/result.py

```diff
@@ -22,6 +23,11 @@ class BlockResult:
     filtered: List[Result] = field(default_factory=list)
     rejected: List[Result] = field(default_factory=list)
 
+    def extend(self, other: 'BlockResult'):
```

Comment:
you can import:
`from __future__ import annotations`

So BlockResult won't be declared with quotes.